web_tool_script_2.R

```diff
@@ -9,12 +9,15 @@ source("0_portfolio_test.R")
 source("0_global_functions.R")
 source("0_web_functions.R")
 
+devtools::load_all()
 setup_project()
 
 working_location <- paste0(working_location, "/")
 
 set_webtool_paths()
 
+# just done once
+options(r2dii_config = paste0(par_file_path,"/AnalysisParameters.yml"))
```

Comment:
For the record, this is a good example to show why #135  is important. 

Here, the variable `par_file_path` is available "magically": It is not visible where it is defined. 
This code should not work, but it does because `set_webtool_paths()` defines `par_file_path` -- beyond
its own scope -- on the global environment. This use of `<<-` is a very bad idea. 

```r
set_webtool_paths <- function() {
  project_location <<- paste0(working_location, "working_dir/")

  log_path <<- paste0(project_location, "00_Log_Files")
  par_file_path <<- paste0(project_location, "10_Parameter_File")
  raw_input_path <<- paste0(project_location, "20_Raw_Inputs")
  proc_input_path <<- paste0(project_location, "30_Processed_Inputs")
  results_path <<- paste0(project_location, "40_Results")
  outputs_path <<- paste0(project_location, "50_Outputs")
}
```